library/ssl_cli.c

```diff
@@ -3301,11 +3301,12 @@ static int ssl_write_client_key_exchange( mbedtls_ssl_context *ssl )
         content_len = own_pubkey_ecpoint_len + 1;
 
         /* Compute ECDH shared secret. */
-        status = psa_key_agreement( &generator,
-                                    handshake->ecdh_psa_privkey,
-                                    handshake->ecdh_psa_peerkey,
-                                    handshake->ecdh_psa_peerkey_len,
-                                    PSA_ALG_ECDH( PSA_ALG_SELECT_RAW ) );
+        status = psa_key_derivation_key_agreement(
+                &generator,
```

Comment:
How would you indent this?

Reply:
```C
status =
    psa_key_derivation_key_agreement(
        &generator,
        handshake->ecdh_psa_privkey,
        handshake->ecdh_psa_peerkey,
        handshake->ecdh_psa_peerkey_len,
        PSA_ALG_ECDH( PSA_ALG_SELECT_RAW ) );
```

But, since this isn't present in the final diff, maybe we don't care enough to fix it.